src/DssSpell.sol

```diff
@@ -20,104 +20,35 @@ pragma solidity 0.6.12;
 //pragma experimental ABIEncoderV2;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import "dss-interfaces/dss/VestAbstract.sol";
 
 import { DssSpellCollateralOnboardingAction } from "./DssSpellCollateralOnboarding.sol";
 
+interface FlashLike {
+    function vat() external view returns (address);
+    function flash() external view returns (address);
+}
 
 contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/6fa5724ffd5801bbcf37802205ed4f54092c6147/governance/votes/Executive%20vote%20-%20March%204%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget  -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-03-04 MakerDAO Executive Spell | Hash: 0x9cf9cccb89d57a94d3b5cb95a5e5df67cdcfaf5310ff0481c194d5cc4a2f3fa2";
-
-    // Recognized Delegates DAI Transfers
-    address constant FLIP_FLOP_FLAP_WALLET  = 0x688d508f3a6B0a377e266405A1583B3316f9A2B3;
-    address constant FEEDBLACK_LOOPS_WALLET = 0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1;
-    address constant ULTRASCHUPPI_WALLET    = 0x89C5d54C979f682F40b73a9FC39F338C88B434c6;
-    address constant MAKERMAN_WALLET        = 0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0;
-    address constant ACRE_INVEST_WALLET     = 0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D;
-    address constant MONETSUPPLY_WALLET     = 0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF;
-    address constant JUSTIN_CASE_WALLET     = 0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473;
-    address constant GFX_LABS_WALLET        = 0xa6e8772af29b29B9202a073f8E36f447689BEef6;
-
-    // Core Units DAI Transfers and Streams
-    address constant CES_001_WALLET         = 0x25307aB59Cd5d8b4E2C01218262Ddf6a89Ff86da;
-    address constant IS_001_WALLET          = 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4;
-
-    // Core Units DAI Streams
-    VestAbstract immutable VEST             = VestAbstract(DssExecLib.getChangelogAddress("MCD_VEST_DAI"));
-
-    address constant RISK_001_WALLET        = 0xb386Bc4e8bAE87c3F67ae94Da36F385C100a370a;
-
-    uint256 constant MAR_01_2022            = 1646092800; // 2022-03-01
-    uint256 constant APR_01_2022            = 1648771200; // 2022-04-01
-    uint256 constant DEC_01_2022            = 1669852800; // 2022-12-01
-    uint256 constant FEB_28_2023            = 1677542400; // 2023-02-28
-    uint256 constant MAR_31_2023            = 1680220800; // 2023-03-31
+        "2022-03-09 MakerDAO Executive Spell | Hash: ";
```

Comment:
Please provide the mitigation contract address to LFW when it's ready and he will add it to the copy.

Reply:
@LongForWisdom `FLASH_KILLER = 0x07a4BaAEFA236A649880009B5a2B862097D9a1cD`